gh-pages/_docs/06-04-tree-map-hierarchy.md

```diff
@@ -0,0 +1,44 @@
+---
+
+permalink: /docs/tree-map-algorithm/
+title: "Tree Map Algorithm"
+
+The CodeCharta map is generated by calculating a map size first as well as placing all 2d buildings and folders as areas. Afterwards we use [three-js](https://threejs.org) to generate the 3d-models at the calculated positions.
+
+In this document we will focus on the first part, the placement algorithm. We use [d3-hierarchy](https://github.com/d3/d3-hierarchy), a module for visualizing hierarchical data, like our code maps. The algorithm operates by placing buildings inside sub-folders and sub-folders inside their parents. All those steps ar performed in a depth-first manner all the way up to the root node.
```

Comment:
Wouldn't a building of size 4 be too small to work with anyways? 
We scale down starting from 40_000 * 40_000 pixels so I don't think it would be recognizable by a user of the tool